x/gov/client/rest/rest.go

```diff
@@ -320,17 +336,32 @@ func queryDepositHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext) http.Han
 		}
 
 		var deposit gov.Deposit
+		// FIXME: We should check the error here but empty/non-existing deposits will
+		// fail to unmarshal.
```

Comment:
Ditto (check error type in switch?)

Reply:
Maybe I should just remove the FIXME all together and leave as-is...idk.

Reply:
Done. Updated.